universum/modules/structure_handler.py

```diff
@@ -221,6 +240,9 @@ def report_background_steps(self) -> bool:
 
     def execute_step_structure(self, configs: Configuration, step_executor) -> None:
         self.configs_total_count = sum(1 for _ in configs.all())
+        for config in configs.all():
+            if config.if_succeeded or config.if_failed:
+                self.configs_total_count += 1
```

Comment:
I think you can just merge this calculation to with the comprehension above.
Something like:
```python
sum(2 if x.if_succeeded or x.if_failed else 1 for x in configs.all())
```

Not sure this would be better, but I prefer the calculation to be uniform.
You can merge the above calculation with your for loop instead.